src/main/java/ch/uzh/ifi/hase/soprafs22/controller/LobbyController.java

```diff
@@ -81,5 +80,22 @@ public void updateLobby(@RequestHeader("token") String token, @PathVariable Long
 
         lobbyService.updateLobby(lobby, token, name, visibility, gameMode, gameType);
     }
+    @GetMapping("/v1/game/lobby")
+    @ResponseStatus(HttpStatus.OK)
+    @ResponseBody
+    public List<LobbyGetDTO> getLobbies() {
+
+        Collection<ILobby> lobbies = lobbyService.getLobbies();
+        List<LobbyGetDTO> lobbiesGetDTOs = new ArrayList();
+        Iterator var4 = lobbies.iterator();
```

Comment:
Use Iterator< ILobby > instead of Iterator and rename it ot lobbiesIterator, lobbiesIt, anything like that but not var4, please

Reply:
ok yes true, thanks!